modules/monitor/apm/topology/topology.go

```diff
@@ -1119,11 +1122,11 @@ func (topology *provider) GetOverview(language i18n.LanguageCodes, params Global
 	}
 	overviewMap["service_count"] = serviceCount
 
-	// 服务实例数
+	// running service instance count
 	instanceMetricsParams := url.Values{}
 	instanceMetricsParams.Set("start", strconv.FormatInt(params.StartTime, 10))
 	instanceMetricsParams.Set("end", strconv.FormatInt(time.Now().UnixNano()/1e6, 10))
-	statement = "SELECT distinct(service_instance_id::tag) FROM application_service_node WHERE terminus_key=$terminus_key GROUP BY service_id::tag"
+	statement = "SELECT service_instance_id::tag,if(gt(now()-timestamp,300000000000),'stopping','running') FROM application_service_node WHERE terminus_key=$terminus_key GROUP BY service_instance_id::tag"
```

Comment:
Did timestamp is not need aggregation in the case of group by？

Reply:
Only need to get the latest timestamp.